app/javascript/packs/components/UserSelect.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react";
+import AsyncSelect from "react-select/lib/Async";
+import { fetch } from "./Fetch";
+export default class UserSelect extends Component {
+  state = {
+    inputValue: ""
+  };
+  getOptionLabel = option => {
```

Comment:
Can be written in one line: `getValue = x => x`, read about auto-return in arrow functions